ThirdParty/PSMTabBarControl/source/PSMYosemiteTabStyle.m

```diff
@@ -410,11 +409,7 @@ - (CGFloat)fontSize {
 #pragma mark - Drawing
 
 - (NSColor *)topLineColorSelected:(BOOL)selected {
-    if (selected) {
-        return [_tabBar.window backgroundColor];
-    } else {
-        return [NSColor colorWithSRGBRed:182/255.0 green:179/255.0 blue:182/255.0 alpha:1];
-    }
+    return [NSColor colorWithSRGBRed:182/255.0 green:179/255.0 blue:182/255.0 alpha:1];
```

Comment:
The OS's tab bar uses a lighter color for the top line of the selected cell. Looks off to me using the same color all the way across.

<img width="486" alt="screen shot 2016-08-24 at 10 35 27 pm" src="https://cloud.githubusercontent.com/assets/427546/17957983/258bb650-6a4b-11e6-9c5c-31cd7f554fcb.png">